services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/resource_tracker_credit_transactions.py

```diff
@@ -50,7 +50,7 @@ async def _sum_credit_transactions_and_publish_to_rabbitmq(
     publish_message = WalletCreditsMessage.construct(
         wallet_id=wallet_id,
         created_at=datetime.now(tz=timezone.utc),
-        credits=wallet_total_credits,
+        credits=wallet_total_credits.available_osparc_credits,
```

Comment:
and here it is actually nicely wrapped in a function.
Why not consider refactoring?

Reply:
Yes in the next iteration, not in this PR which needs to get to Master now. Also lot of additional tests will be coming so it would make sense also to refactor/adjust it for that purpose :+1: